### PR TITLE
Restore default indentation of <dd> items within <dl>

### PR DESCRIPTION
**Pull request by NicolaSaunders**
Proposed solution to https://github.com/w3c/w3c-website/issues/186

### CODE REVIEW

File: assets-src/styles/sass/30-base/_lists.scss [hunk 1]
```diff
@@ -55,6 +55,10 @@ dl {
 		font-weight: bold;
 	}
 
+    dd {
+        margin-inline-start: rem(40);
+    }
```
Comment (vivienlacourba) [inline assets-src/styles/sass/30-base/_lists.scss:60]: If you try on https://design-system.w3.org/code/event.html you will see that it is also going to add some space between each event meta (e.g. between "Date:") and their value.

Before:
image

After:
image

**Review by vivienlacourba — CHANGES_REQUESTED** [review]

File: assets-src/styles/sass/30-base/_lists.scss (hunk 1 above)
Comment (NicolaSaunders) [reply assets-src/styles/sass/30-base/_lists.scss:60]: Apologies for not spotting that - I have pushed a fix for it.

Comment (vivienlacourba) [reply assets-src/styles/sass/30-base/_lists.scss:60]: Thanks. I asked @jean-gui to check this one further so see if there are any other side effects.

**Comment by vivienlacourba** [comment]
@jean-gui can you check this PR further so see if it brings any side effects.

**Review by vivienlacourba — APPROVED** [review]

**Comment by jean-gui** [comment]
This looks ok to me. @NicolaSaunders can you resolve the conflicts it before we can merge it?

**Comment by NicolaSaunders** [comment]
@jean-gui I have resolved conflicts for you now.

**Review by jean-gui — APPROVED** [review]

**Pull request merged**